### PR TITLE
Supply confirmation page set-up - deux

### PR DESCRIPTION
**Pull request by davydka**
Set-up supply confirmation to use patterns from `components/Confirmation`. This page is not wired up to real data yet.
* Moved green checkmark to be inline with confirmation title.
* Card component was only being used within Confirmation components. Made a component `SubRow` with a slightly different organization in order to `vertical-align: baseline;`.
* Modified `FormBuilder` and `Form` to accept accept a subSection that sits above the primary button.

Screenshot 2020-03-29 18 36 25

### CODE REVIEW

File: src/components/Confirmation/Confirmation.styles.js [hunk 1]
```diff
@@ -2,6 +2,10 @@ import { css } from '@emotion/core';
 import { Space } from 'lib/theme';
 
 export const styles = {
+  check: css({
+    marginLeft: Space.S10,
```
Comment (izgeorge) [inline src/components/Confirmation/Confirmation.styles.js]: I think `title` has marginRight: 10 in this file, can you double check we aren't adding margin twice?

File: src/components/Confirmation/SubRow/SubRow.styles.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { css } from '@emotion/core';
+import { Color } from 'lib/theme';
+
+export const styles = {
+  label: css({
+    color: Color.GRAY,
+    fontWeight: 500,
```
Comment (izgeorge) [inline src/components/Confirmation/SubRow/SubRow.styles.js]: do we need this font weight? can we pass `type` to the Text component instead?

File: src/components/Confirmation/SubCta/index.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import { jsx } from '@emotion/core';
+import PropTypes from 'prop-types';
+import Anchor from 'components/Anchor';
+import Text from 'components/Text';
+import { TEXT_TYPE } from 'components/Text/constants';
+import { styles } from './SubCta.styles';
+
+const SubCta = ({ message }) => {
+  return (
+    <Anchor href="/">
```
Comment (izgeorge) [inline src/components/Confirmation/SubCta/index.js]: href should be dynamic

File: src/components/Confirmation/Confirmation.styles.js (hunk 1 above)
Comment (davydka) [reply src/components/Confirmation/Confirmation.styles.js]: 🤘taking it off `.title` (which as getting applied to the container)

File: src/components/Confirmation/SubRow/SubRow.styles.js (hunk 2 above)
Comment (davydka) [reply src/components/Confirmation/SubRow/SubRow.styles.js]: ✂️

File: src/components/Confirmation/SubCta/index.js (hunk 3 above)
Comment (davydka) [reply src/components/Confirmation/SubCta/index.js]: 🎉

**Pull request merged**